apps/front-end/src/v2/components/Static/PageLayout/PageLayout.js

```
@@ -57,11 +65,9 @@ export default function PageLayout({
               showLangChange={showLangChange}
               funLangChange={funLangChange}
             />
-          ) : (
-            <></>
-          )}
+          ) : null}
           {customComponent && <></>}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)